lib/std/meta.zig

```diff
@@ -103,6 +103,38 @@ test "std.meta.stringToEnum" {
     try testing.expect(null == stringToEnum(E1, "C"));
 }
 
+pub fn errorInSet(err: anytype, comptime Set: type) bool {
```

Comment:
Ehh, this function isn't really needed (only used in one place and could just do the check 'by hand').

But, if this would be a useful function more generally, then maybe it is a good idea to keep it in this PR.